src/src/pages/capabilities/KafkaCluster/MessageContracts.js

```diff
@@ -0,0 +1,151 @@
+import styles from "./MessageContract.module.css";
+import Expandable from "../../../components/Expandable";
+import Poles from "../../../components/Poles";
+import { Text } from "@dfds-ui/typography";
+import { SelectField, Switch } from "@dfds-ui/react-components";
+import React, { useEffect, useState } from "react";
+import { MessageHeader, MessageStatus } from "./MessageContract";
+import { Divider } from "@dfds-ui/react-components/divider";
+import SyntaxHighlighter from "react-syntax-highlighter";
+import { vs as syntaxStyle } from "react-syntax-highlighter/dist/esm/styles/hljs";
+
+function JsonViewer({ json }) {
+  return (
+    <div className={styles.messagecontent}>
+      <SyntaxHighlighter
+        language="json"
+        style={syntaxStyle}
+        wrapLongLines={false}
+        customStyle={{
+          margin: "0",
+          padding: "0",
+          border: "1px solid #ccc",
+          height: "370px",
+        }}
+      >
+        {json}
+      </SyntaxHighlighter>
+    </div>
+  );
+}
+
+export default function MessageContracts({
+  id,
```

Comment:
I dont see this set anywhere. Is it leftover from before?

Reply:
I am giving this to the MessageContracts as a param